src/index.html

```diff
@@ -11,8 +12,74 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <!-- #endregion -->
+
+  <body
+    class="card"
+    data-qa="card"
+  >
+    <h1 class="imac">
+      <img
+        src="images/Rectangle 4.png"
+        alt="mac"
+      />
+    </h1>
+
+    <h2 class="h2">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```

Comment:
The class name 'h2' is misleading as it suggests that the element is a heading of level 2, which it is not. Use a more descriptive and appropriate class name.